src/api/listing/taxonomy.proto

```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+package listing_api;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/hdtradeservices/api-proto/go/api/listing";
+
+// Category provides a (usually) high-level classification for a Listing.
+// Categories can be arranged in a tree-like structure with parent and
+// child categories. The category will often be a crucial piece of
+// information about how the data is structured when sending to the
+// channel. For example, on Amazon it dictates which path to take down
+// the XSD when constructing feed messages. An example category on Amazon
+// would be Clothing.
+message Category {
+  // The ID of this category defined by your system
+  string channel_category_id = 1;
+  // References another Category's id as this Category's parent, can be empty
+  // for top-level categories
+  string parent_category_id = 2;
+  // A user-friendly name for the category
+  string display_name = 3;
+
+  google.protobuf.Timestamp updated_at = 4;
+  google.protobuf.Timestamp created_at = 5;
+}
+
+// AttributeSpec is the definition of an attribute on a channel.
```

Comment:
I think it's a little clearer to say that it's the definition of an Attribute within the given Category for a Channel.